code/test/sanityCheck.py

```diff
@@ -12,20 +12,10 @@ def load_yml(yml_file):
     """
     model = cobra.io.load_yaml_model(yml_file)
 
-    # get list of rxns
-    modelRxns = []
-    for r in model.reactions:
-        modelRxns.append(r.id)
-
-    # get list of mets
-    modelMets = []
-    for m in model.metabolites:
-        modelMets.append(m.id)
-
-    # get list of genes
-    modelGenes = []
-    for g in model.genes:
-        modelGenes.append(g.id)
+    # get lists of rxns, mets and genes
+    modelRxns  = list(map(lambda element : element.id, model.reactions))
+    modelMets  = list(map(lambda element : element.id, model.metabolites))
+    modelGenes = list(map(lambda element : element.id, model.genes))
```

Comment:
Nice! How about reusing the _lambda_ function like so:
> fetchId = lambda x : x.id
extractIds = lambda componentArray : list(map(fetchId, componentArray))
[ modelRxns, modelMets, modelGenes] = list(map(extractIds , [model.reactions, model.metabolites, model.genes] ))

Disclaimer: the above might need some testing.

Reply:
this idea was actually considered, but seems against simplicity and readability rules.

Reply:
I see. The first bit though (`fetchId = lambda x : x.id`) should increase readability, unless you expect some variation in the future in how the id will be stored across different component types.